typescript/src/app/newcomp/newcomp.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'te-newcomp',
+  templateUrl: './newcomp.component.html',
+  styleUrls: ['./newcomp.component.css']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/newsbridge/sandbox/pullRequest?prid=7539937)
```suggestion
  styleUrls: ["./newcomp.component.css"]
```